### PR TITLE
[READY]Asay and dsay radio keys

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl: Cyberboss
add: Admins can now asay with ":p" and dsay with ":d"
/:cl:

Yell at me if these are taken

### CODE REVIEW

**Comment by coiax** [comment]
> :p banning AI in AI core

**Comment by ChangelingRain** [comment]
admins that are also aliens won't be able to alien chat!

**Comment by Cheridan** [comment]
why p

**Comment by Cyberboss** [comment]
I dropped a crumb on my keyboard and that's where it landed

**Comment by kevinz000** [comment]
Oh lol I thought you meant adding a physical radio key that lets headsets recieve/send to asay and dsay.

**Comment by kevinz000** [comment]
"hi i'll trade 20 tc for asay key"

**Comment by Cyberboss** [comment]
\> Robust admin
\> Steal radio key
\> Find valids
\> ???
\> Profit

**Comment by optimumtact** [comment]
:p :g absorbing ai

**Comment by lzimann** [comment]
\>not using the verb like a master

File: code/modules/mob/say.dm [hunk 1]
```diff
@@ -2,9 +2,21 @@
 /mob/verb/say_verb(message as text)
 	set name = "Say"
 	set category = "IC"
+
+	if(findtext(message, ":p", 1, 3))
+		if(client)
+			client.cmd_admin_say(trim(replacetext(message, ":p", 1, 3)))
+		return
+
 	if(say_disabled)	//This is here to try to identify lag problems
```
Comment (optimumtact) [inline code/modules/mob/say.dm]: side note, but we really should rip this out now

**Pull request merged**